.github/workflows/cli_test.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build and Test KubeAGI CLI Server
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+env:
+  PYTHON_INDEX_URL: https://pypi.org/simple 
+  PACKAGE_REGISTRY: deb.debian.org
+      
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v4
+          with:
+            fetch-depth: 0
+        - name: Set up QEMU
+          uses: docker/setup-qemu-action@v3
+        - name: Set up Docker Buildx
+          uses: docker/setup-buildx-action@v3
+          with:
+            buildkitd-flags: --debug
+            config-inline: |
+              [worker.oci]
+                max-parallelism = 1
+        - name: Set up GCC
+          uses: egor-tensin/setup-gcc@v1
+          with:
+            version: latest
+            platform: x64
+        - name: Build core library cli image
+          id: push-worker
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: docker/Dockerfile
+            platforms: linux/amd64,linux/arm64
+            push: false
+            tags: test-image
+            build-args: |
+              PACKAGE_REGISTRY=${{ env.PACKAGE_REGISTRY }}
+              PYTHON_INDEX_URL=${{ env.PYTHON_INDEX_URL }}
+        - name: Run Docker Container
+          run: |
+            docker run -d --name temp-container test-image
+            docker ps
+        - name: Run test script
+          run: |
+            sh -C tests/example_test.sh
```

Comment:
It looks like the top part is similar to https://github.com/kubeagi/core-library/blob/main/.github/workflows/image_build_test.yml, so I guess you could just modify that file?

Reply:
I simply cp those into the test action yaml for sure. Could we merge all tests together or leave it here as we might need other tests that would be better executed seperately?